tests/test_github_start_date.py

```diff
@@ -15,17 +16,30 @@ def name():
         return "tap_tester_github_start_date_test"
 
     def test_run(self):
+        self.run_test('2020-04-01T00:00:00Z', '2021-06-10T00:00:00Z', self.expected_streams() - {'events'})
```

Comment:
Please add a comment above, why we expected the event stream.

Reply:
Added comment